### DISCUSSION TITLE
Using config on arch

### DISCUSSION THREAD

**Discussion by trevg22** [Manjaro-Sway/manjaro-sway#430] [Q&A]
I am really enjoying the config and wanted to integrate it into my arch install, will I get most of the features just by cloning desktop-settings and installing the correct packages?

**Comment by boredland** [comment]
sure, most of the magic is in [here](https://github.com/manjaro-sway/desktop-settings/tree/sway/community/sway/). :magic_wand:

**Reply by trevg22** [reply]
Thanks so much, I merged the directories with /etc and /usr/share and I get a waybar_position:Invalid value error, sway must not be using the config. Is there something I need to add to path or add to my dotfiles? I dont see anything else in the desktop settings repo.

**Reply by l-j-g** [reply]
@trevg22 8 months late but ...
`community/sway/etc/skel` should be copied to >> `~/.`
